### DISCUSSION TITLE
Welcome to das-schiff Discussions!

### DISCUSSION THREAD

**Discussion by vukg** [telekom/das-schiff#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robertrebacz** [comment]
Hi Vuk,
I very much liked your presentation today at kubecon. It is impressive to see that you've already decided to use k8s as a foundation for 5G.
I've been wandering which linux distro you use as an operating system for your k8s clusters.
You explained you'd been cooperating with Weavewarks, but I'm not sure if you're using their k8s platform, and anyway it would be great to learn about what you have "underneath" k8s. 
I'm interested because recently there've been a couple of acquisitions changing FOSS OS landscape like redhat to acquire coreOS and then to be acquired by IBM leading to closing coreOS. And very recently MS has acquired kinvolk the creator of flatcar which was suppossed to replace coreOS in the container specialized linux domain.

Thanks for sharing

**Reply by MaxRink** [reply]
Hi @robertrebacz ,

we currently use a minimalized ubuntu as a base image.
We somewhat follow the approach the image-builder project ( https://github.com/kubernetes-sigs/image-builder ) hgas for CAPI Images but currently maintain our own fork of some sorts due to the ammount iof changes we made to harden and minimize the base image.
We have also looked at Flatcar, which looks very promising, although Ingnition support in CAPI isnt fully implimented yet

**Reply by richardcase** [reply]
Hey @robertrebacz  -  for the Liquid Metal demo/proof-of-concept we used ubuntu as our base image. This was mainly as convenience but also because it is used as the base for a lot of Cluster API images. However, going forward on the Weaveworks side we'll be adopting an [image-builder](https://github.com/kubernetes-sigs/image-builder) approach and using OCI images. We're also looking at Flatcar (as @MaxRink mentions CAPI support isn't merged but its very close), Bottlerocket, Clear Linux and potentially something else.

**Reply by robertrebacz** [reply]
Thanks for sharing. Do you have an opinion related to the Rocky project ? I understand this is something "completely different" but would love to hear from you on this. 
Another thing is bare metal vs virtualized nodes for k8s deployments. In your experience - do you have a preference here, or how do you differentiate the use case for bare metal

**Reply by robertrebacz** [reply]
Hello again, sorry to bother, but I have one more question related to Das Schiff directly. In terms of 5G architecture - are 5G core components also deployed on you bare metal k8s clusters ? Or is 5G core deployed separately on some telko specific platform (eg Cisco VIM) , and the "surroundings" are deployed on k8s bare metal clusters.
Again, thanks for sharing

**Reply by Cellebyte** [reply]
@robertrebacz We design our architecture and it should be possible to run the 5G Core on top of our platform directly within our BareMetal Kubernetes clusters.

**Reply by Cellebyte** [reply]
To achieve this we are in deep contact with providers of functions, vendors. hardware distributors and the CNCF CNF WG.
We hope to help serving a use-case for 5G on Kubernetes and moving the industry in this direction.
